chapters/anaconda.Rmd

```diff
@@ -2,7 +2,7 @@
 
 When people first started using Python for data science,
 installing the relevant libraries could be difficult.\index{package installation}
-The main problem was that the Python package installer ([pip][pip])\index{pip}
+The main problem was that the Python package installer ([`pip`][pip])\index{pip}
```

Comment:
I approve the change, but the word nerd in me can't help but notice that we use Git, Python, and Make rather than `git`, `python`, and `make` when referring to the tools. Ah, English...

Reply:
I had exactly the same thought. I think (with the few edits I've made in this PR) we've been consistent with which things are `software` as opposed to Software (i.e. there shouldn't be a mix of conda and `conda` anymore), but I agree that technically Git, Python and Make should be `git`, `python` and `make`. I guess what we've essentially done is use `software` in all cases except for things that we mention frequently in text.